driver/kubernetes/kubernetes.go

```diff
@@ -346,6 +368,7 @@ func generateLabels(op *driver.Operation) map[string]string {
 		"cnab.io/installation": op.Installation,
 		"cnab.io/action":       op.Action,
 		"cnab.io/revision":     op.Revision,
+		"random_string":        strconv.Itoa(rand.Intn(1000)),
```

Comment:
How is this relevant to this PR?

Reply:
It's a temporary fix for a pod race condition I was running into. Sometimes, I ended up streaming previous pods that haven't finished terminating because the labels are the same between trials. I'm currently looking into a better way of doing this, but just wanted to quickly fix the issue.